lit_gpt/config.py

```diff
@@ -82,7 +82,8 @@ def from_name(cls, name: str, **kwargs: Any) -> Self:
     @property
     def mlp_class(self) -> Type:
         # `self._mlp_class` cannot be the type to keep the config json serializable
-        return getattr(lit_gpt.model, self._mlp_class)
+        obj = lit_gpt.lora if (hasattr(self, "mlp_lora") and self.mlp_lora) else lit_gpt.model
+        return getattr(obj, self._mlp_class)
```

Comment:
You could overridde this property in the LoRA config instead: https://github.com/Lightning-AI/lit-gpt/blob/d16f924db91bac43d04ebb91a0a2eb7219af37d9/lit_gpt/lora.py#L383-L397

Reply:
Agree. This approach is much much better. LoRA related changes are now only in LoRA related files.
Done.